src/js/media-time-range.js

```diff
@@ -428,7 +428,7 @@ function getBoxPosition(el, box, percent) {
   // Get the element that enforces the bounding box for the hover preview.
   const mediaBounds =
     (el.getAttribute('media-bounds')
-      ? document.getElementById(el.getAttribute('media-bounds'))
+      ? closestComposedNode(el, `#${el.getAttribute('media-bounds')}`)
```

Comment:
this is because the `media-bounds` references element could be outside the `media-time-range`'s shadow dom?

Reply:
yes, this fixes the case where it was in the shadow dom but `document.getElementById` doesn't have access to it
using closesComposedNode it can get all the ancestors